packages/rich-text/src/RichTextEditor.tsx

```diff
@@ -91,6 +92,9 @@ const getPlugins = (sdk: FieldExtensionSDK, tracking: TrackingProvider) => {
     createCodePlugin(),
     createItalicPlugin(),
     createUnderlinePlugin(),
+
+    // Other
+    createTrailingParagraphPlugin(),
```

Comment:
The order matters here, shouldn't we add it to the top instead? So its rules will run before our plugins, but at the same time I'm afraid our plugins can override some of its rules 🤷‍♀️

Reply:
🤷🏻‍♂️